perftestnotebook.py

```diff
@@ -206,5 +212,40 @@ def main():
         json.dump(ptnb.fmt_data, f, indent=4, sort_keys=True)
 
 
-if __name__ == "__main__":
+    # Upload template file to Iodide
+    template = 'testing/template/template.txt'
+    tdata = ''
+    with open(template, 'r') as f:
+        tdata = f.read()
+
+    html = ''
+    with open('template_upload_file.html', 'r') as f:
+        html = f.read()
+
+    html = html.replace('replace_me', repr(tdata))
```

Comment:
All of this replacement and writing should be done above on line 153 in process.